rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/impl/rosidl_generator_c/message_type_support.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Oh wait, no I don't think this file ever existed as it is here before. I mean it was similar to the header in `rosidl_generator_c` in that it implements the `ROSIDL_*` macros, but each type support package will have a similar header, so it's not a clear moved from here to there situation.